doc/changelog/12-misc/08252-extract_big.rst

```diff
@@ -0,0 +1,9 @@
+- Extraction will now use OCaml's ``big_int``
+
+  OCaml code extracted with the following modules now depends on the ``zarith`` package, and should be linked to it:
```

Comment:
I'm a bit confused by the changelog entry. Would you mind making it more clear what `zarith` is (to me is also a Coq stdlib module). The same for big_int. Maybe a link to these libraries?

Reply:
thanks